src/cow-react/common/containers/FeatureGuard/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { useFeatureFlags } from '@cow/common/hooks/useFeatureFlags'
```

Comment:
Cool you did this FeatureGuard

It would be better to do it as an independent PR (also mention in the PR description)

Reply:
Will do that next time. It was needed to create a component for something like this to make the feature flag possible though.